pipeline/astrometry.py

```diff
@@ -0,0 +1,67 @@
+
+from pipeline.parameters import Parameters
+from pipeline.data_store import DataStore
+
+from models.world_coordinates import WorldCoordinates
+
+
+class ParsAstrometry(Parameters):
+    def __init__(self, **kwargs):
+        super().__init__()
+        self.cross_match_catalog = self.add_par(
+            'cross_match_catalog',
+            'Gaia',
+            str,
+            'Which catalog should be used for cross matching for astrometry. '
```

Comment:
This is fine for here, but I wonder if we might need more information than just a string here.  E.g., now we'd use Gaia DR2.  I don't know if there's a Gaia DR3 planned, but if there were, we might switch to it.

For what goes here, it should just be a string, but that string should then be a key to something somewhere else that actually defines the catalog.  (I.e., there will be catalog grabbing code that uses this string to figure out what it actually gets.)

(So nothing needs to change here, just thinking ahead.)

Reply:
Gaia DR3 is already out since last year... 

Again, we will want to add more parameters to all of these classes, but that will happen naturally when we actually add the code.